### PR TITLE
Highlight Legacy Stackdriver and New Stackdriver for GKE in documentation

### PR DESCRIPTION
**Pull request by pradeepbhadani**

### CODE REVIEW

File: third_party/terraform/website/docs/r/container_cluster.html.markdown [hunk 1]
```diff
@@ -221,7 +221,7 @@ region are guaranteed to support the same version.
     Automatically send metrics from pods in the cluster to the Google Cloud Monitoring API.
     VM metrics will be collected by Google Compute Engine regardless of this setting
     Available options include
-    `monitoring.googleapis.com`, `monitoring.googleapis.com/kubernetes`, and `none`.
+    `monitoring.googleapis.com`(`Legacy Stackdriver`), `monitoring.googleapis.com/kubernetes`(`Stackdriver Kubernetes Engine Monitoring`), and `none`.
```
Comment (slevenick) [inline third_party/terraform/website/docs/r/container_cluster.html.markdown]:
```suggestion
    `monitoring.googleapis.com` (Legacy Stackdriver), `monitoring.googleapis.com/kubernetes` (Stackdriver Kubernetes Engine Monitoring), and `none`.
```

I think this is more clear if the names of the logging endpoints are unquoted

File: third_party/terraform/website/docs/r/container_cluster.html.markdown [hunk 2]
```diff
@@ -184,8 +184,8 @@ making the cluster VPC-native instead of routes-based. Structure is documented
 below.
 
 * `logging_service` - (Optional) The logging service that the cluster should
-    write logs to. Available options include `logging.googleapis.com`,
-    `logging.googleapis.com/kubernetes`, and `none`. Defaults to `logging.googleapis.com/kubernetes`
+    write logs to. Available options include `logging.googleapis.com`(`Legacy Stackdriver`),
+    `logging.googleapis.com/kubernetes`(`Stackdriver Kubernetes Engine Logging`), and `none`. Defaults to `logging.googleapis.com/kubernetes`
```
Comment (slevenick) [inline third_party/terraform/website/docs/r/container_cluster.html.markdown]:
```suggestion
    `logging.googleapis.com/kubernetes` (Stackdriver Kubernetes Engine Logging), and `none`. Defaults to `logging.googleapis.com/kubernetes`
```

File: third_party/terraform/website/docs/r/container_cluster.html.markdown [hunk 3]
```diff
@@ -184,8 +184,8 @@ making the cluster VPC-native instead of routes-based. Structure is documented
 below.
 
 * `logging_service` - (Optional) The logging service that the cluster should
-    write logs to. Available options include `logging.googleapis.com`,
-    `logging.googleapis.com/kubernetes`, and `none`. Defaults to `logging.googleapis.com/kubernetes`
+    write logs to. Available options include `logging.googleapis.com`(`Legacy Stackdriver`),
```
Comment (slevenick) [inline third_party/terraform/website/docs/r/container_cluster.html.markdown]:
```suggestion
    write logs to. Available options include `logging.googleapis.com` (Legacy Stackdriver),
```

Comment (pradeepbhadani) [reply third_party/terraform/website/docs/r/container_cluster.html.markdown]: updated

File: third_party/terraform/website/docs/r/container_cluster.html.markdown (hunk 2 above)
Comment (pradeepbhadani) [reply third_party/terraform/website/docs/r/container_cluster.html.markdown]: updated

File: third_party/terraform/website/docs/r/container_cluster.html.markdown (hunk 1 above)
Comment (pradeepbhadani) [reply third_party/terraform/website/docs/r/container_cluster.html.markdown]: updated

**Pull request merged**